test/integration/toolchains/cmake/test_cmaketoolchain.py

```diff
@@ -1575,3 +1574,53 @@ def _assert_install(out):
     c.run("build .")
     _assert_install(c.out)
     assert "CMAKE_INSTALL_PREFIX" not in c.out
+
+
+def test_toolchain_extra_variables():
+    windows_profile = textwrap.dedent("""
+        [settings]
+        os=Windows
+        arch=x86_64
+        [conf]
+        tools.cmake.cmaketoolchain:extra_variables={'CMAKE_GENERATOR_INSTANCE': '${GENERATOR_INSTANCE}/buildTools/', 'FOO': '42' }
+        """)
+
+    client = TestClient(path_with_spaces=False)
```

Comment:
```suggestion
    client = TestClient()
```
Why? I'd say this test could work with spaces in path

Reply:
Yep, that was copy-paste from test above hehe thanks